Doc/library/http.cookiejar.rst

```diff
@@ -341,6 +345,9 @@ writing.
    compatible with the libwww-perl library's ``Set-Cookie3`` file format.  This is
    convenient if you want to store cookies in a human-readable file.
 
+   .. versionchanged:: 3.8
+
+      The filename parameter supports a :class:`~pathlib.Path` instance.
```

Comment:
As noted below this can be anything that implements __fspath__ and not only pathlib.Path instance. Please use path-like object term with markup (:term:`path-like object`)

Reply:
updated